src/components/input-and-preview.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function InputAndPreview({ input, preview }) {
```

Comment:
Interesting simple wrapper component idea, keeps things cleaner than with mixins or CSS components 👍

Reply:
I also updated `MardownInput` to use this component now.